packages/core/src/test.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  ChatPromptTemplate,
+  PromptTemplate,
+  SimpleDirectoryReader,
+  VectorStoreIndex,
+  type ChatMessage,
+} from "./index.js";
+
+// Define a default prompt template
+const defaultTextQaPromptTemplate = () => `Context information is below.
```

Comment:
does this need to be function?
```suggestion
const defaultTextQaPromptTemplate = "Context information is below.
```